readthedocs/core/management/commands/contact_owners.py

```diff
@@ -143,7 +145,7 @@ def handle(self, *args, **options):
             users = AdminPermission.owners(organization)
         elif usernames:
             file = Path(usernames)
-            with file.open() as f:
+            with file.open(encoding="utf8") as f:
```

Comment:
https://docs.python.org/3/library/locale.html#locale.getpreferredencoding shows this separated by a `-`

```suggestion
            with file.open(encoding="utf-8") as f:
```

But also, this should be the default most of the time, were you having a problem here?

Reply:
Not really. The linter required this. If we don't want to add this to every open() call we can disable it from the linter. I don't have a strong opinion there